src/__a11y__/a11y-page-object.ts

```diff
@@ -57,7 +57,7 @@ export default class A11yPageObject extends BasePageObject {
 
       axe.configure(spec);
 
-      return axe.run(options, runOptions).then(
+      axe.run(options, runOptions).then(
```

Comment:
`this.browser.executeAsync` expects a callback, not a promise. So we should return nothing to follow the typings